client/src/components/_common/LoadingSpinner/LoadingSpinner.scss

```diff
@@ -4,6 +4,7 @@
     align-items: center;
     height: 100%;
     width: 100%;
+    position: absolute;
```

Comment:
Modifying the base loading spinner styling might have unintended consequences for other components that use it. Instead let's apply this styling to the div that contains the spinner, using the preview modal's css modules. You'll find these CSS directives useful: `top: 50%; left: 50%; transform: translate(-50%, -50%)`